test/specs/lib.davIdIcon.spec.js

```diff
@@ -17,4 +17,18 @@ describe('getDavIdIconUrl()', () => {
       return false;  
     }
   };
+
+  test('Return contains the davID within it options',() => {     
+    expect(       
+      checkIfReturnDavId(getDavIdIconUrl(dummyDavId))     
+    ).toBeTruthy();   
+  });    
+  
+  const checkIfReturnDavId = (url) => {     
+    if (url.toLowerCase().indexOf(dummyDavId) >= 0) {       
+      return true;     
```

Comment:
No need to do:
````javascript
if (boolean) {
  return true;
} else {
  return false;
}
````

You can just do:
````javascript
return boolean;
````

Especially since we know the result will be treated as a boolean by `expect().toBeTruthy`